src/CounterSign.cpp

```diff
@@ -462,6 +461,11 @@ bool COSE_Signer_CounterSign_validate(HCOSE_SIGNER hSigner,
 	HCOSE_COUNTERSIGN hCountersignature,
 	cose_errback* perr)
 {
+	if (0) {
```

Comment:
maybe use `false` instead of `0` here and above just to be consequent with the rest below